src/CondenserDotNet.Client/IConfigurationRegistry.cs

```diff
@@ -10,8 +13,10 @@ public interface IConfigurationRegistry
         Task<bool> AddStaticKeyPathAsync(string keyPath);
         Task AddUpdatingPathAsync(string keyPath);
         void AddWatchOnEntireConfig(Action callback);
-        void AddWatchOnSingleKey(string keyToWatch, Action callback);
+        void AddWatchOnSingleKey(string keyToWatch, Action<string> callback);
         Task<bool> SetKeyAsync(string keyPath, string value);
```

Comment:
Looks like you didn't rebase the changes as master updates are coming in, in your commit